vue-app/src/store/index.ts

```diff
@@ -326,14 +315,40 @@ const actions = {
       }
 
       commit(SET_CURRENT_USER, {
+        // By default, when a currentUser has not been loaded before, set all
+        // brightid states to false
+        brightId: {
```

Comment:
But won't this now also overwrite the BrightID state every time `LOAD_USER_INFO` is dispatched (which we trigger every 60 seconds in the App.vue)?

Reply:
I feel like we should just set this to whatever the current state values are.

Reply:
The default `brightId` object then gets overwritten by the spread of `...state.currentUser` (if there is something to spread).

BUT yes...when I did it, I didn't like it either. Honestly, I was just doing it to avoid the optional `brightId?: BrightId` in the interface definition of the User.

I think we could leave it as `brightId?` and clean this ugly thing I did.